specification/storage/resource-manager/Microsoft.Storage/stable/2021-01-01/storage.json

```diff
@@ -0,0 +1,4385 @@
+            "modelAsString": true
+          },
+          "default": "Microsoft.Storage"
+        },
+        "requireInfrastructureEncryption": {
+          "type": "boolean",
+          "x-ms-client-name": "RequireInfrastructureEncryption",
+          "description": "A boolean indicating whether or not the service applies a secondary layer of encryption with platform managed keys for data at rest."
+        },
+        "keyvaultproperties": {
+          "$ref": "#/definitions/KeyVaultProperties",
+          "x-ms-client-name": "KeyVaultProperties",
+          "description": "Properties provided by key vault."
+        },
+        "identity": {
+          "$ref": "#/definitions/EncryptionIdentity",
+          "x-ms-client-name": "EncryptionIdentity",
+          "description": "The identity to be used with service-side encryption at rest."
+        }
+      },
+      "required": [
+        "keySource"
+      ],
+      "description": "The encryption settings on the storage account."
+    },
+    "EncryptionIdentity": {
+      "properties": {
+        "userAssignedIdentity": {
+          "type": "string",
+          "description": "Resource identifier of the UserAssigned identity to be associated with server-side encryption on the storage account.",
+          "x-ms-client-name": "EncryptionUserAssignedIdentity"
+        }
+      },
+      "description": "Encryption identity for the storage account."
+    },
+    "UserAssignedIdentity": {
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal ID of the identity."
+        },
+        "clientId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The client ID of the identity."
+        }
+      },
+      "description": "UserAssignedIdentity for the resource."
+    },
+    "ResourceAccessRule": {
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "Tenant Id"
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "Resource Id"
+        }
+      },
+      "description": "Resource Access Rule."
+    },
+    "VirtualNetworkRule": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "x-ms-client-name": "VirtualNetworkResourceId",
+          "description": "Resource ID of a subnet, for example: /subscriptions/{subscriptionId}/resourceGroups/{groupName}/providers/Microsoft.Network/virtualNetworks/{vnetName}/subnets/{subnetName}."
+        },
+        "action": {
+          "type": "string",
+          "enum": [
+            "Allow"
+          ],
+          "x-ms-enum": {
+            "name": "Action",
+            "modelAsString": false
+          },
+          "default": "Allow",
+          "description": "The action of virtual network rule."
+        },
+        "state": {
+          "type": "string",
+          "enum": [
+            "Provisioning",
```

Comment:
![image](https://user-images.githubusercontent.com/49508232/107309001-a045df00-6ac4-11eb-9fb0-1c8c88222164.png)

This is breaking. Please change back to lowercase.

Reply:
moved back to lowercase.